### ISSUE TITLE
Archive / delete repository?

### ISSUE THREAD

**Issue by vreuter** [pepkit/attmap#78]
@nsheff as we're removing `attmap` dependencies from PEP ecosystem projects, e.g. https://github.com/pepkit/looper/issues/345 , what do you think about marking this project as archived / deprecated...
- [ ] here on Github
- [ ] on PyPI

?

List of repos to remove attmap from before archiving this repo:

- [x] looper
- [x] divvy
- [x] bulker
- [x] peppy
- [x] RefGenConf
- [ ] ...

**Comment by nsheff** [comment]
yes, but probably not for awhile (wait until the dependency is removed from all our projects.)

**Comment by nsheff** [comment]
We can use this issue to keep track of a list of remaining dependencies.

The thing is we need to be aware of both `attmap` and `yacman` < 1.0 dependencies (since yacman < 1.0 will depend on attmap)

**Comment by nsheff** [comment]
I believe everything immediately using attmap has been migrated now. Will leave open a little longer to make sure.
